.github/workflows/test.yaml

```diff
@@ -0,0 +1,26 @@
+name: "Test"
```

Comment:
what would you say it'd be a better name for this workflow? 
`Validate sync diffs`?

Reply:
Agreed